### ISSUE TITLE
Bug: Intrinsic layout doesn't render image in Internet Explorer

### ISSUE THREAD

**Issue by craigscott29** [ampproject/amphtml#17635]
## What's the issue?

When using layout type `intrinsic` with `amp-img`, it results in the error below and the `img` tag does not render within `amp-img`.

> Unable to get property 'setAttribute' of undefined or null reference [object HTMLUnknownElement]

## How do we reproduce the issue?

Here's the set-up code to reproduce the error, I stripped down the `amp-img` example on the AMP Playground so no "outside" code (other than setting the `background-color` to black) would interfere.

```<!doctype html>
<html ⚡>
<head>
  <meta charset="utf-8">
  <title>amp-img</title>
  <script async src="https://cdn.ampproject.org/v0.js"></script>
  <link rel="canonical" href="https://ampbyexample.com/components/amp-img/">
  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
<style amp-custom>
    body {background-color: black;}
</style>
</head>
<body>

  <amp-img src="https://static.craigmanufacturing.com/img/craigLogo_2018.svg" width="421" height="96" layout="intrinsic" alt="AMP"></amp-img>

</body>
</html>
```

Here is a screenshot from the IE dev tools showing that nothing has rendered inside the `amp-img` tag.

![image](https://user-images.githubusercontent.com/33098508/44424609-04286600-a560-11e8-8fe8-a2b3e2eef2c9.png)

## What browsers are affected?

Internet Explorer 11.309.16299.0

## Which AMP version is affected?

1534444305877

**Comment by jpettitt** [comment]
Fixed see #17652

**Comment by AdrianBateman** [comment]
Hi, 

We have the same issue with Internet Explorer version: 11.0.10240.17443 Update: 11.0.43.  

Image is showing as being a very small square instead of the full sized image.

Live Page URL affected:  https://www.dreams.co.uk/amp/duvet-buying-guide

AMP Version: 1810052256480
